executorlib/__init__.py

```
@@ -9,11 +9,19 @@
     "FluxClusterExecutor",
     "SingleNodeExecutor",
     "SlurmJobExecutor",
     "SlurmClusterExecutor",
 ]
+
+try:
+    from executorlib.standalone.hdf import get_cache_data
+except ImportError:
+    pass
+else:
+    __all__ += [get_cache_data]
```

Ruff (0.8.2)

Invalid object in `__all__`, must contain only strings

(PLE0604)